openff/toolkit/_tests/test_molecule.py

```diff
@@ -411,13 +411,6 @@ def test_set_partial_charges_int(self, water):
         with pytest.raises(ValueError, match="Cannot set.*'int'"):
             water.atoms[2].partial_charge = 4
 
-    @requires_pkg("openmm")
-    def test_set_partial_charges_openmm_quantity(self, water):
-        import openmm.unit
-
-        with pytest.raises(ValueError, match="Cannot set.*openmm.unit"):
-            water.atoms[2].partial_charge = 0.0 * openmm.unit.elementary_charge
-
```

Comment:
IIUC, this is no longer an error because the openmm Quantities are now valid inputs.